source/text.c

```diff
@@ -134,11 +141,14 @@ const char* C2D_TextParseLine(C2D_Text* text, C2D_TextBuf buf, const char* str,
 		p += units;
 
 		fontGlyphPos_s glyphData;
-		fontCalcGlyphPos(&glyphData, fontGlyphIndexFromCodePoint(code), 0, 1.0f, 1.0f);
+		C2D_FontCalcGlyphPos(font, &glyphData, C2D_FontGlyphIndexFromCodePoint(font, code), 0, 1.0f, 1.0f);
 		if (glyphData.width > 0.0f)
 		{
 			C2Di_Glyph* glyph      = &buf->glyphs[buf->glyphCount++];
```

Comment:
Alignment is wonky on this section. Remove alignment on both the initial glyph assignment and the if/else block; and readjust the latter xPos/lineNo/width/etc section independently.

Reply:
I think I got this, but feel free to check it again